src/published-data/dto/published-data.obsolete.dto.ts

```diff
@@ -23,6 +63,8 @@ export class PublishedDataObsoleteDto {
   })
   @IsString()
   @IsOptional()
+  @Expose()
+  @Transform(({ obj, key }) => mapPublishedDataV3toV4Field(obj, key))
```

Comment:
maybe it's worth adding `toClassOnly` evereywhere, since that's the only place where the transformation is required